css/clay_css_2_recipes/06_clay_range.md

```diff
@@ -0,0 +1,247 @@
+# Clay Range
+
+This recipe will match Clay Range to [Material Design's Sliders](https://material.io/components/sliders). Clay Range is comprised of several elements. We will mostly be focused on styling `clay-range-track`, `clay-range-progress`, and `clay-range-thumb`.
+
+Material Design specifications state the track should be 4px tall, progress 6px tall, and thumb 20px by 20px. Clay Range uses the mixin [clay-range-input-variant](https://github.com/liferay/clay/blob/0568f0a1ffb82b0bc85321b10cb32ff5f68e2cc1/packages/clay-css/src/scss/mixins/_forms.scss#L540-L909) to generate the styles [here](https://github.com/liferay/clay/blob/0568f0a1ffb82b0bc85321b10cb32ff5f68e2cc1/packages/clay-css/src/scss/components/_range.scss#L31-L33). We can pass styles inside a Sass map through the variable `$clay-range-input`.
+
+_/src/css/\_clay_variables.scss_
+
+```
+$clay-range-input: (
+	thumb-bg: $primary,
+	thumb-box-shadow: 0px 3px 1px -2px rgba(0, 0, 0, 0.2)#{','} 0px 2px 2px 0px rgba(0, 0, 0, 0.14)#{','} 0px 1px 5px 0px rgba(0, 0, 0, 0.12),
+	thumb-height: 1.25rem,
+	thumb-width: 1.25rem,
+	track-bg: $primary-l4,
+	progress-height: 0.375rem,
+);
+```
+
+> One thing to note is the use of [Sass interpolation](https://sass-lang.com/documentation/interpolation) for commas in `thumb-box-shadow`, this is required when passing multiple `box-shadow`s in a Sass map. Interpolating comma prevents Sass from confusing it with the end of a key value pair. This also applies to any CSS property that delimits multiple properties with comma (e.g., `background` and `transition`).
```

Comment:
"Interpolating comma" looks like it should be one of:

- "Interpolating the comma"
- "Interpolating commas"

Similarly, it looks like "that delimits multiple properties with comma" should be "that delimits multilple properites with commas". 🤔?